starter-s3.yaml

```diff
@@ -139,6 +139,9 @@ data:
         - do-not-merge/work-in-progress
         - do-not-merge/invalid-owners-file
 
+      merge_method:
+        kitex-contrib/test-infra: squash
```

Comment:
This doesn't look good to me. none of the merge methods will work here. Since we select all of them as approved type, no matter which one you choose, the result is same.

Reply:
Fine, I'll close this PR.